subprojects/core/src/main/java/org/gradle/StartParameter.java

```diff
@@ -90,6 +81,7 @@
     private boolean continuous;
     private List<File> includedBuilds = new ArrayList<File>();
     private boolean buildScan;
+    private String bootstrapInitScriptUrl;
```

Comment:
I wouldn't add another property but  reuse the existing list. Just like you can define properties both on the cli and in a properties file, you'd be able to define init script(s) in both places.